doc/cug.tex

```diff
@@ -5821,25 +5871,41 @@ \subsection{Suite And Task Event Handling}
 See also~\ref{SuiteEventHandling} and~\ref{TaskEventHandling}
 in the {\em Suite.rc Reference}.
 
-Cylc can call nominated event handlers when certain suite or task events
-occur. This is intended to facilitate centralized alerting and automated
-handling of critical events. Event handlers can send an email or an SMS,
-call a pager, and so on; or intervene in the operation of their own
-suite using cylc commands.  \lstinline=cylc [hook] email-suite= and
-\lstinline=cylc [hook] email-task= are example event handlers packaged
-with cylc.
+Cylc can call nominated event handlers when certain suite or task events occur.
+This is intended to facilitate centralized alerting and automated handling of
+critical events. Event handlers can be used to send a message, call a pager,
+and so on; or intervene in the operation of their own suite using cylc
+commands.
 
-Event handlers can be located in the suite bin directory, otherwise
-it is up to you to ensure their location is in \lstinline=$PATH=
-(in the shell in which cylc runs, on the suite host).
-
-Task event handlers are passed the following arguments by the suite daemon:
+To send an email, you can use the built-in setting
+\lstinline=[[[events]]]mail events= to specify a list of events when
+notifications should be sent. E.g. to send an email on (submission) failed and
+retry, you can do:
 
+\lstset{language=suiterc}
 \begin{lstlisting}
-<task-event-handler> EVENT SUITE TASKID MESSAGE
+[runtime]
+    [[foo]]
+        retry delays = PT0S, PT30S
+        script = test "${CYLC_TASK_TRY_NUMBER}" -eq 3
+        [[[events]]]
+            mail events = submission failed, submission retry, failed, retry
```

Comment:
`cylc email-suite` is still useful until we have made the same change for suite event handlers.

I guess we can deprecate `cylc email-task`, but removing it will cause suites using it to fail. (We don't use it at our site, but I am sure other users do?)